e2e_tests/compute/instance.go

```diff
@@ -47,50 +45,6 @@ func (i *Instance) Cleanup() {
 	}
 }
 
-// WaitForSerialOutput waits to a string match on a serial port.
-func (i *Instance) WaitForSerialOutput(match string, port int64, interval, timeout time.Duration) error {
```

Comment:
is this not being used anymore? iirc, this is what we use for test assertions

Reply:
Yeah, we dropped it in favor of using guest attributes